search/includes/classes/class-search.php

```diff
@@ -1188,8 +1188,8 @@ public function ep_handle_failed_request( $request, $response, $query, $type, $c
 			return;
 		}
 
-		if ( ! $is_cli ) {
-			global $wp;
+		global $wp;
```

Comment:
Should this be placed at the top of this method? I kept it here only to preserve the current style.